src/wp2hugo/internal/hugogenerator/hugopage/wordpress_youtube_converter.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Example: Plain-text Youtube URLs on their own line in post content are turned by WP into embeds
-var _YoutubeRegEx = regexp.MustCompile(`(?m)(^|\s)https?://(?:m\.|www\.)?(?:youtu\.be|youtube\.com)/(?:watch|w)\?v=([^&\s]+)`)
+var _YoutubeRegEx = regexp.MustCompile(`(?m)(^|\s)http[sav]?://(?:m\.|www\.)?(?:youtu\.be|youtube\.com)/(?:watch|w)\?v=([^&\s]+)`)
```

Comment:
Nit: Please add a reason for `httpa` and `httpv` and a link to https://wordpress.com/plugins/wp-youtube-lyte in the comments above _YoutubeRegEx

Reply:
Done!